packages/flutter_test/lib/src/binding.dart

```diff
@@ -96,12 +97,17 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   ///
   /// This constructor overrides the [debugPrint] global hook to point to
   /// [debugPrintOverride], which can be overridden by subclasses.
-  TestWidgetsFlutterBinding() {
+  // TODO(mattcarroll): find a way to use super.window instead of ui.window
```

Comment:
i wouldn't worry about it, using ui.window is fine. If it becomes a problem we can revisit it. No need to add a TODO, IMHO.